index.bs

```diff
@@ -1643,14 +1636,22 @@ Constructors</h4>
         1. Let |document| be the [=current settings object=]'s [=relevant global object=]'s
                 [=associated Document=].
 
+        1. Let |validationResult| be the return value of
+                <a href="#validating-sink-identifier">sink identifier validation
+                </a> of {{AudioContext/[[sink ID]]}}.
+
+        1. If |validationResult| is not <code>true</code>, [=Queue a
```

Comment:
`not <code>true</code>` 

to

`<code>false</code>`

Reply:
Done, thanks!